projects/commudle-admin/src/app/feature-modules/labs/components/labs/search-bar/search-bar.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="search-container">
-  <div (clickOutside)="toggleSearchSuffix(false)" appClickOutside class="lab-search" (mouseleave)="toggleSearchSuffix(false)">
```

Comment:
When we are searching tags, if I move my mouse out of the lab-search div then that div gets hidden. Also, if we are using the mouse leave event then the click outside isn't needed.

I wasn't sure which one to use because both aren't needed at the same time. Any suggestions?

Reply:
I think let's add animation to the dropdown (as in smooth open and close), and let the mouse leave from the complete div (including search bar and list of searched tags)
Outside click might be needed on mobile (not sure if it works without it, if it does, then we should remove the outside click handler)

Reply:
Also, have pushed some changes to the feature-labs-revamp branch, please do merge them with your code :)

Reply:
> I think let's add animation to the dropdown (as in smooth open and close), and let the mouse leave from the complete div (including search bar and list of searched tags)
> Outside click might be needed on mobile (not sure if it works without it, if it does, then we should remove the outside click handler)

Okay I'll check


> Also, have pushed some changes to the feature-labs-revamp branch, please do merge them with your code :)

Sure